available-connectors.json

```diff
@@ -668,6 +668,30 @@
     },
     "supportedByStrongLoop": false
   },
+  {
+    "name": "twilio",
+    "description": "Twilio Connector",
+    "baseModel": "Model",
```

Comment:
I am not very familiar with twilio, usually if the data is persistently stored on the back end, the `baseModel` should be `PersistedModel`.

(Ignore my comment if you understand the difference between the 2 base models and choose `Model` with a good reason.)